src/vst_plugin/RnNoiseVstPlugin.h

```diff
@@ -21,8 +21,29 @@ class RnNoiseVstPlugin : public AudioEffectX {
 
     void getProgramName(char *name) override;
 
+    void getParameterLabel(VstInt32 index, char* label) override;
+
+    void getParameterName(VstInt32 index, char* label) override;
+
+    void getParameterDisplay(VstInt32 index, char* label) override;
+
+    float getParameter(VstInt32 index) override;
+
+    void setParameter(VstInt32 index, float value) override;
+
+    static const VstInt32 numParameters = 1;
+
 private:
     static const char* s_programName;
 
+    enum class Parameters {
+        vadThreshold = 0
+    };    
+    
+    // Parameter: VAD Threshold
+    const char* paramVadThresholdLabel = "";
+    const char* paramVadThresholdName = "VAD Threshold";
+    float paramVadThreshold;
```

Comment:
I'm not sure if it will be always initialized by `setParameter`, so could you make it `float paramVadThreshold{0.f};`?